docs/running-aggregate-with-docker.md

```diff
@@ -0,0 +1,57 @@
+# Running Aggregate with Docker
+
+## Requirememts
+
+- Install [Docker](https://www.docker.com)
+
+## Quick start - External PostgreSQL instance
+
+- Build the Docker image with `./gradlew clean dockerBuild -xtest -pwarMode=complete`.
+
+  Take note of the tag of the created image. It will appear in a message like this one:
+  
+  ```
+  ...
+  Successfully built bf09fb92a6e9
+  Successfully tagged aggregate:v1.6.0-beta.0-dirty
+  ```
+
+- Launch Aggregate for the first time with `docker run -d -p 8080:8080 -it --name=aggregate aggregate:v1.6.0-beta.0-dirty`.
+ 
+  Replace the version with the one you got when building the image
+
+  Access Aggregate at [http:/localhost:8080](http:/localhost:8080)
```

Comment:
s|http:/|http://

Reply:
This makes it seem like you'll get Aggregate loading at that URL. You won't right? You'll need to make sure the external DB is configured and working first.